frontend/web/components/pages/ProjectSettingsPage.js

```diff
@@ -604,9 +604,28 @@ const ProjectSettingsPage = class extends Component {
                       />
                     </TabItem>
                     <TabItem tabLabel='Custom Fields'>
-                      <MetadataPage
-                        organisationId={AccountStore.getOrganisation().id}
-                        projectId={this.props.match.params.projectId}
+                      <Row space className='mb-2 mt-4'>
+                        <Row>
+                          <h5>Custom Fields</h5>
+                        </Row>
+                      </Row>
+
+                      <WarningMessage
+                        warningMessage={
+                          <span>
+                            This tab is now located in the Organisation Settings
+                            page.{' '}
+                            <a
+                              href={`/organisation/${
+                                AccountStore.getOrganisation()?.id
+                              }/settings?tab=custom-fields`}
+                              rel='noreferrer'
+                            >
+                              Click here to go there
+                            </a>
+                            .
```

Comment:
I think we can be slightly less verbose here. 

```suggestion
                            Custom fields have been moved to {' '}
                            <a
                              href={`/organisation/${
                                AccountStore.getOrganisation()?.id
                              }/settings?tab=custom-fields`}
                              rel='noreferrer'
                            >
                              Organisation Settings
                            </a>
                            .
```